src/entities/player.lua

```diff
@@ -50,31 +71,48 @@ function player:update(ground, dt)
 
   self.joints = {}
 
-  -- setup keyboard event handling
   inertia = self.body:getInertia()
-
-  angle = self.body:getAngle()
   mass = self.body:getMass()
-  if love.keyboard.isDown("w") then
-    if love.keyboard.isDown("a") then
-      self.body:applyTorque(-self.turnMultiplier * inertia)
-    elseif love.keyboard.isDown("d") then
+
+  vx, vy = self.body:getLinearVelocity()
+  ax = (vx - self.vx) / dt
+  ay = (vy - self.vy) / dt
```

Comment:
These should be local along with some variables previously defined in this function.